### DISCUSSION TITLE
How to force execute queries for dashboards? Adding `maxAge=0` does not help

### DISCUSSION THREAD

**Discussion by sullerandras** [getredash/redash#6467] [Q&A]
My goal is to ensure that when a dashboard is opened then it shows the latest available data (instead of cached data from hours or days ago). Adding the `maxAge=0` query parameter forces that particular query to refresh but how to achieve the same for all queries on a dashboard?

**Comment by sullerandras** [comment]
It seems the redash ui makes POST https://redash.mydomain.com/api/queries/180/results api requests for all queries on the dashboard with payload like this: `{"id":180,"parameters":{"mydate":{"start":"2023-09-01","end":"2023-09-30"}},"apply_auto_limit":false}`

It does return cached data.

However if i add `"max_age":0` in the payload then it returns a new job, and i assume the frontend would handle that and load the refreshed data.
So how to add this `max_age` parameter to the api requests? Neither `&maxAge=0` nor `&max_age=0` helps.

**Reply by gaecoli** [reply]
`max_age: 0` is worked!

**Reply by sullerandras** [reply]
Hi @gaecoli could you clarify? Is there a solution now to add `"max_age": 0` to the json payload?

**Reply by gaecoli** [reply]
`{"id":180,"parameters":{"mydate":{"start":"2023-09-01","end":"2023-09-30"}},"apply_auto_limit":false, "max_age": 0}`
